client/ayon_core/pipeline/create/context.py

```diff
@@ -1987,12 +1987,16 @@ def create(
                     "Folder '{}' was not found".format(folder_path)
                 )
 
-        task_name = None
         if task_entity is None:
             task_name = self.get_current_task_name()
-            task_entity = ayon_api.get_task_by_name(
-                project_name, folder_entity["id"], task_name
-            )
+            if task_name:
+                task_entity = ayon_api.get_task_by_name(
+                    project_name, folder_entity["id"], task_name
+                )
+
+        task_name = None
+        if task_entity:
+            task_name = task_entity["name"]
```

Comment:
Would this be clearer?
```suggestion
        task_name = task_entity["name"] if task_entity else None
```

Reply:
Or even better yet - thus set that directly [here](https://github.com/ynput/ayon-core/pull/480/files#diff-aef2d4c691a5a8a08e92e98a60998b7636c34882b238c8805b45f34c8ffd7cf5R2025) so that `"task": task_entity["name"] if task_entity else None` to make it clearer it's only used there?